frontend/mr-admin-flate/src/components/tiltaksgjennomforinger/TiltaksgjennomforingSkjemaDetaljer.tsx

```diff
@@ -311,17 +313,60 @@ export const TiltaksgjennomforingSkjemaDetaljer = ({ tiltaksgjennomforing, avtal
                     />
                   </div>
                 )}
-              <TextField
-                size="small"
-                label="Sted for gjennomføring"
-                description="Skriv inn stedet tiltaket skal gjennomføres, for eksempel Fredrikstad eller Tromsø. For tiltak uten eksplisitt lokasjon (for eksempel digital jobbklubb), kan du la feltet stå tomt."
-                {...register("stedForGjennomforing")}
-                error={
-                  errors.stedForGjennomforing
-                    ? (errors.stedForGjennomforing.message as string)
-                    : null
-                }
-              />
+              {isTiltakSomKreverFremmoteData(avtale.tiltakstype.arenaKode) ? (
+                <>
+                  <Separator />
+                  <Label>Fremmøte</Label>
+                  <div
+                    style={{
+                      display: "flex",
+                      flexDirection: "row",
+                      gap: "1rem",
+                    }}
+                  >
+                    <ControlledDateInput
+                      label="Dato"
+                      readOnly={erArenaOpphav(tiltaksgjennomforing)}
+                      fromDate={minStartdato}
+                      toDate={maxSluttdato}
+                      size="small"
+                      {...register("fremmoteDato")}
+                      format="iso-string"
+                    />
+                    <TextField
+                      size="small"
+                      type="time"
+                      style={{
+                        width: "100px",
+                      }}
+                      readOnly={erArenaOpphav(tiltaksgjennomforing)}
+                      error={errors.fremmoteTid?.message as string}
+                      {...register("fremmoteTid")}
+                      label="Klokkeslett"
+                    />
+                  </div>
+                  <Textarea
```

Comment:
Kan du legge på litt høyde med rows-propertyen her, og evt. bruke cols-propertyen for å redusere bredden noe. Tekstfeltet ble veldig bredt og smalt nå.